### DISCUSSION TITLE
What is the capital of 🇪🇸?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#788] [Galaxy Trivia] [answered]
**Bonus:** What is the capital of BAT? 🐧

**Comment by peterbaccaro** [comment, marked as answer]
The capital of Spain is **Madrid**.
The capital of the British Antarctic Territory is **Rothera**.

**Comment by SANKARAMDAS** [comment]
Madrid, Spain 🇪🇸 Madrid is the capital and most populous city of E-Spain.
